frontend/components/Dashboard/PostCard/PostCard.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import Link from 'next/link'
+
+interface IPostCardProps {
```

Comment:
In other typescript/React apps, I've just done `interface Props {` since it's local to the file and isn't used elsewhere. What do ya think?

Reply:
It's quite a common convention to prefix interface names with `I` so I was just establishing that across the app, but I suppose you're right that in the case of Props it might not matter as much since it's local and not used elsewhere. We can go with that for `Props` 👍🏼✅